GoogleDataTransport/GDTCCTLibrary/GDTCCTUploadOperation.m

```diff
@@ -38,7 +38,6 @@
 #import "GoogleDataTransport/GDTCCTLibrary/Private/GDTCCTNanopbHelpers.h"
 #import "GoogleDataTransport/GDTCCTLibrary/Private/GDTCCTURLSessionDataResponse.h"
 #import "GoogleDataTransport/GDTCCTLibrary/Private/GDTCOREvent+GDTMetricsSupport.h"
```

Comment:
Can this category (or any other) cause a similar issue?

Reply:
Possibly so I've added the force linking C function trick to force link them all.